oracle/src/oracleimporter.cpp

```diff
@@ -33,7 +33,7 @@ bool OracleImporter::readSetsFromByteArray(const QByteArray &data)
 
     while (it.hasNext()) {
         map = it.next().toMap();
-        edition = map.value("code").toString();
+        edition = map.value("code").toString().toUpper();
```

Comment:
That's not needed anymore. See https://github.com/mtgjson/mtgjson/pull/196.

Is it worth removing this again?
Would it help with quicker parsing the json and creating our database? @ctrlaltca